website_v2/gcloud-powershell-website-v2/www/static/templates/cmdlet.html

```diff
@@ -12,6 +12,13 @@
     </pre>
     <parameter-table ng-if="property === 'parameters'"></parameter-table>
   </div>
+  <!-- Displays all related links for a cmdlet -->
+  <div ng-show="!mainCtrl.isEmpty(mainCtrl.productInfo[mainCtrl.params.product][mainCtrl.params.cmdlet].links)">
```

Comment:
`mainCtrl.productInfo[mainCtrl.params.product][mainCtrl.params.cmdlet].links` is pretty crazy. Why don't you define a function to get the product info?

e.g. `mainCtrl.getInfo("links")` or similar.